### ISSUE TITLE
Missing backpropagation method

### ISSUE THREAD

**Issue by jihwanK** [darksigma/Fundamentals-of-Deep-Learning-Book#9]
I ran this code, but I found that this model wasn't properly trained at all. I think, in this 6th block, the code may require a proper backpropagation method (loss.backward()), doesn't it? Unless you intended it to be so. This made me quite confused for quite long.

<img width="1416" alt="image" src="https://github.com/darksigma/Fundamentals-of-Deep-Learning-Book/assets/17960812/a238f601-48ac-4e71-a75d-c9b867732bb6">
